src/main/java/frc/robot/commands/Zeroing/ManualZeroElevator.java

```diff
@@ -67,6 +68,7 @@ public void end(boolean interrupted) {
     if (!interrupted) {
       globalElevator.hasZeroed = true;
       globalElevator.resetSensorPosition(constElevator.ZEROED_POS);
+      globalElevator.setCoastMode(false);
```

Comment:
Is this the best spot for this? Should we always do this in end or just when its not interrupted? how does it get set back if it doesnt finish? also what happens if the robot is mid elevation and goes into disable?